Sofa/Component/Topology/Container/Dynamic/src/sofa/component/topology/container/dynamic/TriangleSetTopologyContainer.cpp

```diff
@@ -125,6 +125,13 @@ void TriangleSetTopologyContainer::createTrianglesAroundVertexArray()
     m_trianglesAroundVertex.resize(getNbPoints());
     for (size_t i = 0; i < m_triangle.size(); ++i)
     {
+        if (m_triangle[i][0] >= getNbPoints() || m_triangle[i][1] >= getNbPoints() || m_triangle[i][2] >= getNbPoints())
+        {
+            msg_warning() << "trianglesAroundVertex creation failed, Triangle buffer is not concistent with number of points: Triangle: " << m_triangle[i] << " for: " << getNbPoints() << " points.";
+            continue;
+        }
```

Comment:
It could yes! I just copy-pasted (included the typo on consistent!) what was done from EdgeSetTopologyContainer where there is exactly the same loop but for edges.

Reply:
what? My télor is riche! 😆